### ISSUE TITLE
Incompatible with Bamboo Everything mod.

### ISSUE THREAD

**Issue by K0-RR** [Micalobia/Full-Slabs#13]
![image](https://user-images.githubusercontent.com/43277609/114297446-0218ca00-9ab1-11eb-93cb-52615efdbe47.png)

**Comment by Micalobia** [comment]
Should be fixed in 2.0.1, which should come out soon

**Comment by K0-RR** [comment]
Out of the curiosity - why these slabs weren't working? What differs them from slabs from other mods? I'm going to check if it's working now and close this.

**Comment by Micalobia** [comment]
In texture models, specific textures are usually labeled with `#top` or `#all`, etc. in the templates, then other textures would derive from those. With how this mod builds textures, it has to get each texture by those specific names and sub them into a different one, so I had to hard-code a few of em to get it to work with vanilla. Most mods also use the same templates that vanilla does, but Bamboo Everything used a specialized thing, so I had to hard-code their values into it as well.

It isn't the best system, but it's the best I've come up with without lagging things out quite a bit.
